tests/src/testlinearelasticity.cpp

```diff
@@ -32,110 +36,111 @@ int main(int argc, char** argv) {
-                          Dune::ReferenceElements<double, 3>::cube(), linearElasticFunc3D, Ikarus::skills(),
-                          Ikarus::AffordanceCollections::elastoStatics, checkGradientFunctor, checkHessianFunctor,
+                          Dune::ReferenceElements<double, 3>::cube(), linearElasticFunc3D, skills(),
+                          AffordanceCollections::elastoStatics, checkGradientFunctor, checkHessianFunctor,
                           checkJacobianFunctor, checkFEByAutoDiffFunctor));
   t.subTest(testFEElement(secondOrderLagrangePrePower3Basis, "LinearElastic", randomlyDistorted,
-                          Dune::ReferenceElements<double, 3>::cube(), linearElasticFunc3D, Ikarus::skills(),
-                          Ikarus::AffordanceCollections::elastoStatics, checkGradientFunctor, checkHessianFunctor,
+                          Dune::ReferenceElements<double, 3>::cube(), linearElasticFunc3D, skills(),
+                          AffordanceCollections::elastoStatics, checkGradientFunctor, checkHessianFunctor,
                           checkJacobianFunctor, checkFEByAutoDiffFunctor));
   t.subTest(testFEElement(secondOrderLagrangePrePower3BasisBlocked, "LinearElastic", randomlyDistorted,
-                          Dune::ReferenceElements<double, 3>::cube(), linearElasticFunc3D, Ikarus::skills(),
-                          Ikarus::AffordanceCollections::elastoStatics, checkGradientFunctor, checkHessianFunctor,
+                          Dune::ReferenceElements<double, 3>::cube(), linearElasticFunc3D, skills(),
+                          AffordanceCollections::elastoStatics, checkGradientFunctor, checkHessianFunctor,
                           checkJacobianFunctor, checkFEByAutoDiffFunctor));
 
   t.subTest(testFEElement(
       firstOrderLagrangePrePower3Basis, "LinearElastic", unDistorted, Dune::ReferenceElements<double, 3>::cube(),
-      linearElasticFunc3D, Ikarus::skills(), Ikarus::AffordanceCollections::elastoStatics,
-      checkCalculateAtFunctorFactory<Ikarus::ResultTypes::linearStress>(linearStressResultsOfCube),
-      checkResultFunctionFunctorFactory<Ikarus::ResultTypes::linearStress>(linearStressResultsOfCube),
-      checkResultFunctionFunctorFactory<Ikarus::ResultTypes::linearStress, Ikarus::ResultEvaluators::VonMises>(
-          linearVonMisesResultsOfCube),
-      checkResultFunctionFunctorFactory<Ikarus::ResultTypes::linearStress,
-                                        Ikarus::ResultEvaluators::PrincipalStress<3>>(
-          linearPrincipalStressResultsOfCube)));
+      linearElasticFunc3D, skills(), AffordanceCollections::elastoStatics,
+      checkCalculateAtFunctorFactory<linearStress>(linearStressResultsOfCube),
+      checkResultFunctionFunctorFactory<linearStress>(linearStressResultsOfCube),
+      checkResultFunctionFunctorFactory<linearStress, VonMises>(linearVonMisesResultsOfCube),
+      checkResultFunctionFunctorFactory<linearStress, PrincipalStress<3>>(linearPrincipalStressResultsOfCube)));
```

Comment:
It is also perhaps nice to have tests for the newly added result evaluators in 3D. What do you think?